.goreleaser.yml

```diff
@@ -12,11 +12,10 @@
 # - macOS (only 10.15 is supported)
 # - amd64
 
-
 builds:
   - id: std
     ldflags:
-      - -X main.VERSION={{.Version}} -X main.commit={{.FullCommit}} -X main.buildDate={{.Date}} -X main.builtBy=goreleaser
+      - -X github.com/TykTechnologies/tyk-pump/pumps.VERSION={{.Version}} -X github.com/TykTechnologies/tyk-pump/pumps.commit={{.FullCommit}} -X github.com/TykTechnologies/tyk-pump/pumps.buildDate={{.Date}} -X github.com/TykTechnologies/tyk-pump/pumps.builtBy=goreleaser
```

Comment:
For this repo, it actually appears to be on the `main` package :/
https://github.com/TykTechnologies/tyk-pump/blob/master/version.go

Also `commit` is actually `Commit` (similar in other repos it seems).